packages/dso-toolkit/src/components/modal/modal.mixins.scss

```diff
@@ -160,25 +191,32 @@
     }
 
     .dso-footer {
-      min-height: auto;
+      min-block-size: auto;
       overflow: auto;
+
       padding: modal-variables.$footer-padding-small-modal-vh modal-variables.$footer-padding;
     }
   }
+
+  // html/css version fullscreen on =< 480px viewport
```

Comment:
Geldt dit alleen voor de HTML/CSS version?

Reply:
Ja, omdat de mixin met een andere argument wordt aangeroepen. Maar dat kleien we dus om.